src/find_github_repo.ipynb

```diff
@@ -518,7 +518,7 @@
    "source": [
     "from github import Github\n",
     "\n",
-    "g = Github(\"MelanieIStefan\", \"hn8PJuM\")\n"
```

Comment:
This is unfortunately still present in the history since you made a commit with it. I bet you can [close out](https://github.com/blog/1509-personal-api-tokens) this key though so it can't be used in the future...